bin/setup

```diff
@@ -5,7 +5,6 @@ def setup!
     test "ruby -v" => ruby_version
     run  "gem install bundler --no-document --conservative"
     run  "bundle install"
-    run  "bin/yarn install" if File.exist?("yarn.lock")
```

Comment:
@G-Rath This creates an issue when running ci-run.sh and previously I remember it was complaining that there was no bin/yarn file.

I cannot figure out why this line was for. Do you know if there is anything else I can do to test this change?

Did you you want me to put it back and try to get it working?

Reply:
It's because `bin/yarn` no longer exists, which is fine - we can replace it with just `yarn install`.